Makefile.emscripten

```diff
@@ -167,20 +169,19 @@ ifeq ($(HAVE_AL), 1)
    DEFINES += -DHAVE_AL
 endif
 
-ifneq ($(PTHREAD), 0)
-   LDFLAGS += -s WASM_MEM_MAX=1073741824 -pthread -s PTHREAD_POOL_SIZE=$(PTHREAD)
+ifeq ($(PTHREAD), 1)
```

Comment:
See comment above about HAVE_THREADS, which seems to determine whether we want -lpthread or not already.